libjlm/src/opt/unroll.cpp

```diff
@@ -432,7 +472,6 @@ unroll_unknown_theta(const unrollinfo & ui, size_t factor)
 			smap.insert(*olv, xv);
 		}
 	}
-
```

Comment:
Noise, please reinsert.

Reply:
So you like meaningless whitespaces :)

Reply:
Fair enough. Leave it as it is now. Thanks!